Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use an official Python runtime as a parent image
+FROM python:3.9-slim-buster
```

Comment:
I think python 3.11.X is out. Can you update the image? We typically keep on the latest version(s). Once a month we try to upgrade everything and unless something is broken we usually upgrade to the latest version or a release behind the latest.